docs/user/image_mode.md

```diff
@@ -0,0 +1,517 @@
+
+# Lock root user account
+rootpw --lock
+
+# Configure network to use DHCP and activate on boot
+network --bootproto=dhcp --device=link --activate --onboot=on
+
+%pre-install --log=/dev/console --erroronfail
+
+# Create a 'bootc' image registry authentication file
+mkdir -p /etc/ostree
+cat > /etc/ostree/auth.json <<'EOF'
+$(cat "${AUTH_CONFIG}")
+EOF
+
+%end
+
+# Pull a 'bootc' image from a remote registry
+ostreecontainer --url "${IMAGE_REF}"
+
+%post --log=/dev/console --erroronfail
+
+# Create an OpenShift pull secret file
+cat > /etc/crio/openshift-pull-secret <<'EOF'
+$(cat "${PULL_SECRET}")
+EOF
+chmod 600 /etc/crio/openshift-pull-secret
+
+%end
+EOFKS
+```
+
+The kickstart file uses a special [ostreecontainer](https://pykickstart.readthedocs.io/en/latest/kickstart-docs.html#ostreecontainer)
+directive to pull a `bootc` image from the remote registry and use it to install
+the RHEL operating system.
+
+### Create Virtual Machine
+
+Download a RHEL boot ISO image from https://developers.redhat.com/products/rhel/download.
+Copy the downloaded file to the `/var/lib/libvirt/images` directory.
+
+Run the following commands to create a RHEL virtual machine with 2 cores, 2GB of
+RAM and 20GB of storage. The command uses the kickstart file prepared in the
+previous step to pull a `bootc` image from the remote registry and use it to install
+the RHEL operating system.
+
+```bash
+VMNAME=microshift-4.18-bootc
+NETNAME=default
+
+sudo virt-install \
+    --name ${VMNAME} \
+    --vcpus 2 \
+    --memory 2048 \
+    --disk path=/var/lib/libvirt/images/${VMNAME}.qcow2,size=20 \
+    --network network=${NETNAME},model=virtio \
+    --events on_reboot=restart \
+    --location /var/lib/libvirt/images/rhel-9.4-$(uname -m)-boot.iso \
+    --initrd-inject kickstart.ks \
+    --extra-args "inst.ks=file://kickstart.ks" \
+    --wait
+```
+
+Log into the virtual machine using the `redhat:<password>` credentials.
+Run the following command to verify that all the MicroShift pods are up and running
+without errors.
+
+```bash
+watch sudo oc get pods -A \
+    --kubeconfig /var/lib/microshift/resources/kubeadmin/kubeconfig
+```
+
+## Using Bootc Image Builder (BIB)
+
+The [bootc-image-builder](https://github.com/osbuild/bootc-image-builder), is a
+containerized tool to create disk images from bootc images. You can use the tool
+to generate various image artifacts and deploy them in different environments,
+such as the edge, server, and clouds.
+
+### Create ISO image using BIB
+
+```bash
+PULL_SECRET=~/.pull-secret.json
+IMAGE_NAME=microshift-4.18-bootc
+
+mkdir ./output
+sudo podman run --authfile ${PULL_SECRET} --rm -it \
+    --privileged \
+    --security-opt label=type:unconfined_t \
+    -v /var/lib/containers/storage:/var/lib/containers/storage \
+    -v ./output:/output \
+    registry.redhat.io/rhel9/bootc-image-builder:latest \
+    --local \
+    --type iso \
+    localhost/${IMAGE_NAME}:latest
```

Comment:
Are we back in this step to 'build host' (from the hypervisor)? Because that image ref suggest the build host but it's not explicitly told to go back.